src/components/Badge/BadgeConstants.ts

```diff
@@ -0,0 +1,19 @@
+import { DialogPosition } from "../../constants";
+
+export enum BadgeType {
+  INDICATOR = "indicator",
+  COUNTER = "counter"
+}
+
+export enum BadgeAnchor {
+  TOP_START = DialogPosition.TOP_START,
+  TOP_END = DialogPosition.TOP_END,
+  BOTTOM_START = DialogPosition.BOTTOM_START,
+  BOTTOM_END = DialogPosition.BOTTOM_END
+}
+
+export enum BadgeAlignments {
+  CORNER = "corner",
+  OUTSIDE = "outside",
+  EMBEDDED = "embedded"
+}
```

Comment:
Please share your thoughts on the naming, after a brainstorm with Yael this are the best we could come up with

Reply:
I really prefer the previous approach. Maybe round, rectangle and transparent?